typespec-tests/tsp/visibility.tsp

```diff
@@ -18,6 +18,11 @@ model Dog {
   name: string;
 }
 
+model RoundTripModel {
+  name: string;
+  @visibility("create", "update") secretName: string;
```

Comment:
I assume this regard same as a `x-ms-secret` for Swagger (when visibility contains create or update, but not contain read).